src/completer/model.ts

```diff
@@ -45,7 +45,7 @@ class CompleterModel implements CompleterWidget.IModel {
     return this._original;
   }
   set original(newValue: CompleterWidget.ITextState) {
-    if (Private.deepEqual(newValue, this._original)) {
+    if (JSONExt.deepEqual(newValue, this._original)) {
```

Comment:
We need to assert that the values are not `undefined` for this to work with the new version, that's why I added the private function.